nexus/src/authz/actor.rs

```diff
@@ -0,0 +1,93 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Oso integration for Actor types
+
+use super::roles::RoleSet;
+use crate::authn;
+use omicron_common::api::external::ResourceType;
+use uuid::Uuid;
+
+/// Represents [`authn::Context`] (which is either an authenticated or
+/// unauthenticated actor) for Polar
+#[derive(Clone, Debug)]
+pub struct AnyActor {
+    authenticated: bool,
+    actor_id: Option<Uuid>,
+    roles: RoleSet,
+}
```

Comment:
Why is this called `AnyActor` instead of `Actor`?

Reply:
Just to better distinguish it from `AuthenticatedActor`.

(That predates this change -- this code just moved.)